doc/operation-and-maintenance/System-Metrics-Privacy-Policy.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+MongooseIM system metrics are gathered to analyse the trends and needs of our users, improve MongooseIM, and know where to focus our efforts.
+This section is devoted to explaining on how customise, read, enable and disable collecting of the system metrics.
+
+# Consent
+To ensure the transparency of the functionality being enabled, a log message is generated on every MongooseIM node start (unless metrics service is configured with `report` option).
+The user is being notified that the metrics are gathered and has the right to withdraw the consent at any time without limiting functionalities of the product.
+For more information on how to disable this feature, please see the [Services](Services.md#service_mongoose_system_metrics) section.
```

Comment:
I think that link doesn't work, try it [there](https://github.com/esl/MongooseIM/blob/documentation_system_metrics/doc/operation-and-maintenance/System-Metrics-Privacy-Policy.md) 🤔 
You need to put the path relative to the current file, so something like
```suggestion
For more information on how to disable this feature, please see the [Services](../advanced-configuration/Services.md#service_mongoose_system_metrics) section.
```
should do.